### PR TITLE
Warn when suspecting absence of .cro.yml

### PR DESCRIPTION
**Pull request by Altai-man**
Closes https://github.com/croservices/cro/issues/115

### CODE REVIEW

File: lib/Cro/Tools/CLI.pm6 [hunk 1]
```diff
@@ -296,6 +296,9 @@ sub run-services(:$filter = *, :$trace = False, :@trace-filters, :$host) {
                     note $data.trim.indent($prefix.chars + 2);
                 }
             }
+            when Cro::Tools::Runner::PossiblyNoCroConfig {
+                note colored("Did not encounter a single service whlie scanning the directory for $_.timing() seconds, could it be .cro.yml file is missing?", "bold");
```
Comment (jnthn) [inline lib/Cro/Tools/CLI.pm6:300]: Maybe reword it as "Did not encounter any cro.yml files while scanning the directory "$*CWD"`. I'm not sure the timing bit is really helpful. (Maybe more robust to include the directory in the exception rather than assume $*CWD though).

Comment (Altai-man) [reply lib/Cro/Tools/CLI.pm6]: Addressed.

**Review by jnthn — APPROVED** [review]
